scripts-circleci/github-package-helm-dispatch.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 VERSION_FILE=${VERSION_FILE:="/workspace/version.txt"}
-GITHUB_WORKFLOW=${GITHUB_WORKFLOW:="qr_package-helm-chart.yaml"}
+GITHUB_WORKFLOW="qr_pipeline.yaml"
```

Comment:
Possibly, the script should be renamed from github-package-helm-dispatch.sh to github-pipeline-dispatch.sh. But then we should maybe generalize the script so that it can trigger qr_pipeline.yaml in the general case, i.e. with VERSION either empty or non-empty.